transit/models/order_details.py

```diff
@@ -29,11 +29,15 @@ def without_shipment_details(self):
         return self.get_queryset().without_shipment_details()
 
 
+def generate_order_id():
```

Comment:
I wonder whether should we move this into 'utils' etc.

Reply:
I think it can stay here